client/landing/gutenboarding/components/header/style.scss

```diff
@@ -58,22 +58,6 @@ $padding--gutenboarding__header-domain-picker-button: 0.3em 0.5em;
 
 	&:disabled {
 		opacity: 1;
-		.gutenboarding__site-title {
-			color: black;
-		}
-	}
-
-	.gutenboarding__site-title {
-		padding: 0;
-	}
-
-	> span {
-		display: flex;
-		flex-direction: column;
-		align-items: flex-start;
-	}
-	> .dashicon {
```

Comment:
Same - need to double check if any longer needed.

Reply:
The icon is also used here (assuming `DashIcon` component uses it internally): https://github.com/Automattic/wp-calypso/blob/update/gutenboarding-header-styles/client/landing/gutenboarding/components/domain-picker-button/index.tsx#L80